### ISSUE TITLE
I want to add a new card for fonts section

### ISSUE THREAD

**Issue by PriyansuMaurya** [rupali-codes/LinksHub#31]
### Description

I have a doubt about whether to create a new file for it or continue with the current one [ `../../database/data `] we already have.

https://fontjoy.com/

**Comment by rupali-codes** [comment]
yes, u can

**Comment by PriyansuMaurya** [comment]
> yes, u can

should I create a new file?

**Comment by rupali-codes** [comment]
No, u doesn't need to
just add an object in data array (under database/data.ts file)

example:
{
    id: +1 to previous element,
    name: <site name>,
    description: <site desc>,
    url: <url>,
    category: "frontend", //since it belongs to frontend
    subcategory: "fonts", //since its a font's website
  }

**Comment by Ankit5kumar** [comment]
Could assign  me task

**Comment by rupali-codes** [comment]
@Ankit5kumar  thanks for taking interest. You can create another issue then pull request.

**Comment by Ankit5kumar** [comment]
Could I u tell me
How to u want to authenticate a user from backend 
I create the backend for the authentication
Do I need to write the code in TypeScript?

**Comment by rupali-codes** [comment]
@Ankit5kumar Hi, no you don't have to. Just go to issues tab then new issue and you will get Add Link issue template [here](https://github.com/rupali-codes/LinksHub/issues/new/choose), otherwise just go to [database/data.ts](https://github.com/rupali-codes/LinksHub/blob/main/database/data.ts) under root of the project and add object of values similar to previous objects. 

Let us know for any help

**Comment by rupali-codes** [comment]
> Could I u tell me How to u want to authenticate a user from backend I create the backend for the authentication Do I need to write the code in TypeScript?

You can just links to resources which belongs to authentication

**Comment by Ankit5kumar** [comment]
Error: Could not find a production build in the 'D:\typeScript\LinksHub\.next' directory. Try building your app with 'next build' before starting the production server. https://nextjs.org/docs/messages/production-start-no-build-id

I am getting the above error
while npm start

**Comment by rupali-codes** [comment]
run `next build`

**Comment by Ankit5kumar** [comment]
λ  (Server)  server-side renders at runtime (uses getInitialProps or getServerSideProps)
○  (Static)  automatically rendered as static HTML (uses no initial props)

what does it mean?

**Comment by Ankit5kumar** [comment]
I clone the project from main branch But is showing different output from live project
Should I clone it from the dev branch ?

**Comment by PriyansuMaurya** [comment]
> I clone the project from the main branch But is showing different output from the live project Should I clone it from the dev branch?

don't clone bro use codespaces instead. it would be much easier.

**Comment by Ankit5kumar** [comment]
ok
